index.html

```diff
@@ -0,0 +1,268 @@
+        <img src="images/bubble-9.png" class="bubble9">
+      </div>
+      <div class="container_top_conent">
+        <div class="baner_text">
+          <div class="baner_title">Design</div>
+          It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum.
+        </div>
+      </div>
+    </div>
+    <!-- ######################################### ABOUT SECTION ######################################### -->
+    <div class="container_about">
+      <div class="left">
+        <h7>About Us</h7>
+        <div class="aboutUs_title">Why branding matters?</div>
+        <div class="about_text">
+          It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
+        </div>
+        <div class="list_select">
+          <img src="images/strategy.png" class="strategy">
+          Strategy
+          <div class="list_text">Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model and web page editors</div>
+        </div>
+        <div class="list">
+          <img src="images/design.png" class="design">
+          Design
+        </div>
+        <div class="list">
+          <img src="images/earth.png" class="earth">
+          Development
+        </div>
+      </div>
+      <div class="right">
+        <div class="form_container">
+          <div class="form_title">Get a free Keystroke <br /> quote now</div>
+          <div class="form_name">Name</div>
+          <div class="form_inbox">John Smith</div>
+          <div class="form_name">Email</div>
+          <div class="form_inbox">example@mail.com</div>
+          <div class="form_name">Phone</div>
+          <div class="form_inbox">+123456789</div>
+          <div class="form_button">Get Pricing Now</div>
+        </div>
+      </div>
+    </div>
+    <!-- ######################################### DESIGN SECTION ######################################### -->
+    <div class="design_container">
+      <div class="design_background">
+        <img src="images/bubble-24.png" class="bubble24">
+        <img src="images/bubble-23.png" class="bubble23">
+        <img src="images/line-4.png" class="line_process line_discover">
+        <img src="images/line-5.png" class="line_process line_prototype">
+        <img src="images/line-4.png" class="line_process line_test">
+      </div>
+      <div class="design_content">
+        <div class="design_title">
+          <h7>Process</h7>
+          <h1>Our logo design process</h1>
+          <div>
+            <h3>Our comprehensive logo design strategy ensures a perfectly crafted logo for your business.</h3>
+          </div>
+        </div>
+        <div class="element">
+          <div class="photo"><img src="images/process-1.png" class="photo_image"></div>
+          <div class="element_content">
+            <h7>Step One</h7>
```

Comment:
Jakie to ma znaczenie dla wyszukiwarki, że to jest pierwszy krok? Dla mnie to nieistotne więc nie powinno być nagłówkiem. To samo w innych miejscach.